### PR TITLE
RSBTB-3392 added purgeContentElement method to the translation table

### PR DESCRIPTION
**Pull request by Mathewlenning**
First set to resolve RSBTB-3392

### CODE REVIEW

File: extensions/libraries/redcore/translation/table.php [hunk 1]
```diff
@@ -1206,4 +1206,25 @@ public static function setInstalledTranslationTables($option, $table, $contentEl
...
 	}
+
+	/**
+	 * Method to purge contents of a translation table
+	 *
+	 * @param   string  $tableName  name of either the translations table or the table being translated I.E. #__content or #__content__rctranslations
+	 *
+	 * @return bool
+	 */
+	public static function purgeContentElement($tableName)
```
Comment (Kixo) [inline extensions/libraries/redcore/translation/table.php]: you know that you have purge table by ID now? https://github.com/Mathewlenning/redCORE/blob/RSBTB-3392-Fatal-Error-When-Resetting-Translation-Tables/extensions/libraries/redcore/translation/table.php#L780

Comment (Mathewlenning) [inline extensions/libraries/redcore/translation/table.php]: I didn't dig that deep. Just implemented the missing method used by redshopb. Should I just use that method instead?

Comment (Kixo) [inline extensions/libraries/redcore/translation/table.php]: yeah... use $translationTables = RTranslationTable::getInstalledTranslationTables(true);
then in the loop just use this function...
RTranslationTable::purgeTables($translationTable->id);

**Comment by Mathewlenning** [comment]
@Kixo I implemented it as you suggested, so I'm closing this PR. Thanks =^D

**Pull request closed without merging**